corehq/apps/domain/templates/domain/partials/bootstrap3/new_stripe_card_template.html

```diff
@@ -6,48 +6,66 @@
     <div class="modal-dialog">
       <div class="modal-content">
         <div class="modal-header">
-          <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+          <button
+            type="button"
+            class="close"
+            data-dismiss="modal"
+            aria-label="Close"
+          >
+            <span aria-hidden="true">&times;</span>
+          </button>
           <h4 class="modal-title">{% trans "Add New Credit Card" %}</h4>
         </div>
         <div class="modal-body form-horizontal">
-          <input type="hidden"
-                 name="stripeToken"
-                 data-bind="value: token" />
-          <div class="alert alert-success"
-               data-bind="visible: isProcessing">
-            <i class="fa fa-refresh fa-spin"></i> {% trans "Processing your request" %}
-          </div>
-          <div class="alert alert-danger"
-               data-bind="visible: errorMsg() != '',
-                                    text: errorMsg">
+          <input type="hidden" name="stripeToken" data-bind="value: token" />
+          <div class="alert alert-success" data-bind="visible: isProcessing">
+            <i class="fa fa-refresh fa-spin"></i> {% trans "Processing your
+            request" %}
```

Comment:
I don't like that Prettier is splitting template tags across multiple lines like this. I wonder if it's an issue during parsing or formatting.